Adds a Space Gym Vault

Arthurdentist opened the pull request:
This PR adds a new vault to the game, the 'gym'.

![gym ingame](https://cloud.githubusercontent.com/assets/6353372/13822247/cf75308e-eb9c-11e5-9c59-67ff66913f24.png)

The gym has two unique 'gym loot' spawners and a standard maint loot spawner. It also has a unique Darkholme suit named 'jabroni outfit' as well as a pill bottle full of creatine 'gym supplements' There is an issue with the pills which I need to resolve before it can be merged but that shouldn't be too tricky. The gym is protected by three unique zombies; Leatherman, Boss of this Gym and Boy Next Door. 

This isn't finished yet and i'm looking for suggestions for loot and other things. Its loot is pretty weak compared to the other vaults but im not sure what else to add, if anything.

Kitfox88 commented:
fucking amazing :100: :+1:

ihadtoregisterforthis commented:
Two leather suits?

Boxe commented:
but is it leatherman or leatherhead?

the answer is both

unid15 commented:
Make the leather***** have a 50% chance of being named leatherman and 50% chance of being named leatherhead

DrCelt commented:
It's leatherman though

Boxe commented:
sounds like leathermeh to me

Arthurdentist commented:
@ihadtoregisterforthis Just the one. You can't remove the one the zombie is 'wearing'.

KingDairy commented:
can we get a dungeon master skull cap while we're at it

ihadtoregisterforthis commented:
That's terrible

TheEliminator16 commented:
The real question is how far away is the leather club

Arthurdentist commented:
@ihadtoregisterforthis there is no way I'm coding in the ability for zombies to wear clothes just for this.

TheEliminator16 commented:
>gay porn memes find their way into spessmen

I love this server

ihadtoregisterforthis commented:
No homo

Boxe commented:
@TheEliminator16

well most of the admins are gay so it's no surprise

Arthurdentist commented:
Fixed the issue with the creatine pills and added two more loot spawners as well as a couple of decals to make it look more ruined. 

![gym updated](https://cloud.githubusercontent.com/assets/6353372/13834632/8d1fdc60-ebe8-11e5-95f1-807457ae6577.png)

Intigracy commented:
@Arthurdentist no need to code it in specially, look at how russians handle it.

Exxion commented:
This is a good label

clusterfack commented:
Your memes end here

FudgePucker commented:
:+1: 
Now all you got to do is add a leather club vault.

Arthurdentist commented:
this is ready to merge unless someone really wants me to change how the leatherman zombie works. Suppose I could also changelog but I hear its broken.

NigglyWiggly commented:
You should change the leatherman zombie to have the correct name

KingDairy commented:
someone should just stream lords of the locker room so we can settle this in the ring

PJB3005 commented:
What the fuck is this shitty pretending to be [Honk] tag.

Arthurdentist commented:
merge when

9600bauds commented:
It's not the Honk tag, it's for memes.

Arthurdentist commented:
does the memes tag have a stigma?

This is the only thing I have ever seen tagged with it.

Exxion commented:
Because it's the only thing tagged with it

Arthurdentist commented:
gee whizz

TheEliminator16 commented:
Really the "Leather***" mob needs to randomly pick "man" and "head".
You wouldn't want to inaccurately portray a very divisive issue, would you?

Exxion commented:
>Cluster "NO FUN ALLOWED" Fack

Didn't you also remove the Usain Bolt label?

clusterfack commented:
Labels aren't there to be fun or entertaining, they are meant to organize issues and PRs for coders to effectively find them so we can do our collaborating more easily.

It's the same reason I mock shitty bug reports that spend 80% of their text to make fun of coders or spout memes instead of reporting issues.

Exxion commented:
But [Usain Bolt] was added *by Pomf himself*

Also explain [Honk] then

Intigracy commented:
Honk is the one joke label that's permitted.

TheEliminator16 commented:
What about [OH GOD IT'S LOOSE]

Probe1 commented on maps/randomvaults/spacegym.dmm:
```diff
@@ -0,0 +1,44 @@
+"p" = (/obj/item/weapon/stool,/turf/simulated/floor/airless,/area/vault/gym)
+"q" = (/obj/map/spawner/misc/gym,/turf/simulated/floor/airless,/area/vault/gym)
+"r" = (/mob/living/simple_animal/hostile/necro/zombie{name = "boy next door"},/turf/simulated/floor/airless,/area/vault/gym)
+"s" = (/obj/item/clothing/under/darkholme{name = "jabroni outfit"},/turf/simulated/floor/airless,/area/vault/gym)
```
:+1:  :100:

Probe1 commented:
also yeah what clusterfack said, don't go batty with labels please

clusterfack commented:
@TheEliminator16 Those are high priority bugs

Exxion commented:
Admittedly, [OH GOD IT'S LOOSE] is a serious tag with a silly name
But there's also [IT'S A FEAAAATUUUUREEEEEEEEE]

Intigracy commented:
It'd serve the same purpose if it said "Feature" or "It's a Feature".

It's just made more witty.

TheEliminator16 commented:
I actually didn't know that [OH GOD IT'S LOOSE] was meant to be serious, learn something new every day.

9600bauds commented:
>they are meant to organize issues

Someday someone will try to research the precedence of memes and will be mildly inconvenienced

Intigracy commented:
string up bauds by his bauds

ihadtoregisterforthis commented:
HONK tag is about jokes, memes aren't a joke

ihadtoregisterforthis commented:
https://youtu.be/yS8P4km0m3k

unid15 commented on vgstation13.dme:
```diff
@@ -1932,6 +1932,7 @@
 #include "maps\defficiency\areas.dm"
 #include "maps\defficiency\pipes.dm"
 #include "maps\randomvaults\objects.dm"
+#include "maps\randomvaults\spacegym.dmm"
```
Shouldn't be here! This adds a 7th zlevel to the map

Intigracy replied:
the one above it too, I assume?

NigglyWiggly replied:
Ruh roh

Intigracy replied:
already fixed

NigglyWiggly replied:
Then wouldn't it say something about it being an outdated version?
EDIT: Nevermind, just saw #8908

Intigracy replied:
This was merged before it was fixed.

I made a seperate PR that fixed this right after reading unid's comment.
